include/nfc/ndef/msg_parser_local.h

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Can this be removed from includes and added as internal include in nfc subsys?

Reply:
If yes, we don't have to be that strict in terms of naming conventions in this file

Reply:
It is possible. Changed